src/gnssro/gnssro_bufr2ioda.f90

```diff
@@ -194,34 +194,26 @@ program gnssro_bufr2ioda2
          cycle read_loop
      endif
 
-!    profile check:  (1) CDAAC processing - cosmic-1, cosmic-2, sacc, cnofs, kompsat5
-     if ( ((said >= 740).and.(said <=745)).or.((said >= 750).and.(said <= 755)) &
-            .or.(said == 820).or.(said == 786).or.(said == 825) &
-            .or. ogce == 60) then  !CDAAC processing
-       if(pcc == 0.0) then
+     if(pcc == 0.0) then
           write(6,*)'READ_GNSSRO: bad profile 0.0% confidence said=',said,'ptid=',ptid, ' SKIP this report'
           cycle read_loop
-        endif
      endif
 
      bendflag = 0
      refflag  = 0
-!    profile check:  (2) GRAS SAF processing - metopa-c, oceansat2, megha-tropiques, sacd
-     if ( (said >= 3 .and.said <= 5).or.(said == 421).or.(said == 440).or. (said == 821) ) then
-          call upftbv(lnbufr,nemo,qfro,mxib,ibit,nib)
-          if(nib > 0) then
-            do i = 1, nib
-               if(ibit(i)== 5) then  ! bending angle
-                  bendflag = 1
-                  write(6,*)'READ_GNSSRO: bad profile said=',said,'ptid=',ptid, ' SKIP this report'
-                  cycle read_loop
-               endif
-               if(ibit(i)== 6) then  ! refractivity
-                  refflag = 1
-                  exit
-               endif
-           enddo
-         endif
+     call upftbv(lnbufr,nemo,qfro,mxib,ibit,nib)
+     if(nib > 0) then
+        do i = 1, nib
+            if(ibit(i)== 5) then  ! bending angle
```

Comment:
The python script checks bit 1, 4, 5 but Fortran code here only checks 5, which might cause discrepancy?

Reply:
@haydenlj Was this fixed?

Reply:
I was talking about Ibit=5, which is for the bending angle. I was talking about matching the python code checks
` if i_non_nominal != 0 or i_phase_non_nominal != 0 or i_bang_non_nominal != 0:
        return {}` 
in the python converter. The code here only checks  i_bang_non_nominal (ibit 5), but not the other two

Reply:
I understand now, sorry. It's fixed

Reply:
@huishao-r are you good with these updates, ie is it okay to merge? Thanks!